blazor/pdfviewer/how-to/change-the-highlighted-color-of-the-text.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Change the text highlight color in Blazor PDF Viewer | Syncfusion
+description: Learn here all about how to change the highlighted color of the text in Syncfusion Blazor PDF Viewer component.
+platform: Blazor
+control: PDF Viewer
+documentation: ug
+---
+
+# Change the highlight color of the text in Blazor PDF Viewer Component
+
+You can change the highlight color of the selected annotation using the `Color` property of the `PdfViewerHighlightSettings` class.
```

Comment:
Please provide the hyper link for this class

Reply:
Provided the hyperlink.